pages/common/flutter.md

```diff
@@ -0,0 +1,23 @@
+# flutter
+
+> Google’s free and open source cross-platform mobile app SDK.
+
+- Check the Flutter version:
+
+`flutter --version`
+
+- Call general help:
+
+`flutter help`
+
+- Call help on a command:
```

Comment:
Perhaps `Display help about a specific command` would read better?

Reply:
That's right, that sounds better.